src/modules/analysis/AnalysisAddComponent.js

```diff
@@ -142,6 +143,42 @@ const AnalysisAdd = (
     <PageHeader style={{ paddingLeft: 20 }}>
       {I18n.t('musit.analysis.registeringAnalysis')}
     </PageHeader>
+    <Form horizontal style={{ paddingLeft: 10 }}>
+      <FormGroup>
+        <Label label="Type analyse" md={1} />
+        <Col md={2}>
+          <select
+            className="form-control"
+            onChange={updateFormField(form.analysisTypeCategory, updateForm)}
+          >
+            <option>Velg kategori</option>
+            {uniq(store.analysisTypes.map(b => b.category)).map(a => (
```

Comment:
We are using the existing data-set twice. We need all AnalysisTypes to show. So we use all AnalysisTypes to get different categories to select specific AnalysisTypes.